src/Files.App/ViewModels/UserControls/StatusCenterViewModel.cs

```diff
@@ -92,7 +110,7 @@ public void RemoveAllCompletedItems()
 		{
 			for (var i = StatusCenterItems.Count - 1; i >= 0; i--)
 			{
-				if (!StatusCenterItems[i].IsInProgress)
+				if (!StatusCenterItems[i].IsExpandable)
```

Comment:
```suggestion
				if (!StatusCenterItems[i].IsSpeedAvailable)
```

Reply:
Not only speed but also progress persentage and so I decided to use Expandable.

Reply:
What about IsSpeedAndProgressAvailable?

Reply:
Should be that such detailed?

Reply:
I think it's helpful in this case

Reply:
Done

Reply:
I assume this would still work with the preparing stage for operations?

Reply:
- Unsupported tasks like Decompressing and Compressing
- Preparing tasks

So IsExpandable was good but it was up to @yaira2.

Reply:
Thanks

Reply:
IsExpandable is  scoped to the control, so it makes sense as its a control functional property - if you are handling things on the Data end, then properties like ``isSpeedAvailable`` or ``isInProgress`` makes sense.

Ideally you would do something like 
```
if(iStatusCardItem.isSpeedAvailable = true && iStatusCardItem.isInProgress = true)
{
    StatusCard.IsExpandable = true;
}
else
{
    StatusCard.IsExpandable = false;
}
```